File: october/25-10/co-de.py

def complementary_dna(strand):

    DICTIONARY = {
        "A" : "T",
        "T" : "A",
        "C" : "G",
        "G" : "C"
    }

    complementary = ""
    for char in strand:
        complementary += DICTIONARY[char]
    
    return complementary

File: october/25-10/code.js
// start of script.js **

function complementaryDNA(strand) {

    const dictionary = new Map();
    dictionary.set("A", "T");
    dictionary.set("T", "A");
    dictionary.set("C", "G");
    dictionary.set("G", "C");

    let complementary = "";
    for(let char of strand) {
        complementary += dictionary.get(char);
    }

    return complementary;
}

// end of script.js **

